eth/vm/forks/constantinople/constants.py

```diff
@@ -2,5 +2,11 @@
 
 
 GAS_EXTCODEHASH_EIP1052 = 400
+GAS_SSTORE_NOOP_EIP1283 = 200
```

Comment:
It'd be ideal if the `EIP1283` namespace was located before the parts that are different.

```python
GAS_SSTORE_EIP1283_INIT = ...
GAS_SSTORE_EIP1283_CLEAN = ...
```

Makes the namespace more effective imho.